ckan-2.10/base/Dockerfile.py-3.10

```
@@ -0,0 +1,92 @@
+# use the official Python 3.10 (bookworm slim) image as the base image
+FROM python:3.10-slim-bookworm
+
+# Tag passed through via the Makefile
+ARG CKAN_TAG=${CKAN_TAG}
+
+ENV TZ=UTC
+ENV APP_DIR=/srv/app
+ENV SRC_DIR=${APP_DIR}src
```

```suggestion
ENV SRC_DIR=${APP_DIR}/src
```

This is putting stuff in /srv/appsrc instead of /srv/app/src